src/ol_openedx_rapid_response/README.rst

```diff
@@ -0,0 +1,67 @@
+ol_openedx_canvas_integration
+=============================
+
+Overview
+--------
+
+A django app plugin for edx-platform to enable ``Rapid Response`` reports download functionality in ``Instructor`` tab.
```

Comment:
Question: is this plugin going to be expanded to implement the actual rapid response functionality? In other words, is the `rapid-response-xblock` functionality going to be moved into here, then we'll get rid of that repo? 

If yes, we should update this overview and various comments in this PR to be less specific. Something like "A django app plugin for edx-platform to enable ``Rapid Response`` problems in Open edX". If we don't do it now, we will almost definitely forget to do it later

If not, and this plugin is _only_ going to be used for the report functionality, then this plugin should be renamed to something more specific like `ol-openedx-rapid-response-reports`, and various other things like folder/file names and comments should be updated in the same way

Reply:
This plugin would not be expanding with the Rapid Response xBlock functionality. That xBlock can be installed and used independently but the reports download functionality is optional which can now be installed as a plugin alongside the rapid response if needed.
_I've updated the plugin name as per your suggestion_